apps/amp-als/dataset-service/src/main/java/org/sagebionetworks/amp/als/dataset/service/api/DatasetApi.java

```diff
@@ -111,7 +111,7 @@ default DatasetApiDelegate getDelegate() {
     value = "/datasets/{datasetId}",
     produces = { "application/json", "application/ld+json", "application/problem+json" }
   )
-  default ResponseEntity<DatasetDto> getDataset(
+  default ResponseEntity<?> getDataset(
```

Comment:
Workaround to support returning both JSON and JSON-LD payloads (JSON-LD support will be added in an upcoming PR).
Note: This file is overwritten whenever the OpenAPI generator is run. A more robust solution would be to use a custom template. Additionally, the use of `?` should be replaced with a more descriptive type.